prow/oss/tests/README.md

```diff
@@ -0,0 +1,7 @@
+# prow/oss/tests
+
+This directory contains tests for the jobs deployed on [oss-prow.knative.dev](https://oss-prow.knative.dev/)
+
+These tests enforce a number of project-specific conventions.
+
+To run via go: `go test .`
```

Comment:
What makes sure these run on presubmit?

Reply:
You mean this job? https://github.com/GoogleCloudPlatform/oss-test-infra/blob/004bf03d9f22193c91ec46c731bfd083349c5583/prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml#L23

Reply:
Since it run `go test ./...` everything under root gets tested

Reply:
What is the plan for getting these set up with bazel?